WeCount/datapackage.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "COVID-19-testing-locations-WeCount",
+	"datapackage_version": "0.1.0",
+	"title": "WeCount project consolidated COVID-19 assessment centre locations in Ontario",
+	"description": "Contain data files of COVID-19 assessment centre locations in Ontario. These files are consolidated used by the WeCount project.",
+	"keywords": ["Ontario", "Health", "COVID-19", "coronavirus", "COVID", "testing", "COVID-19 testing locations", "WeCount Project", "Inclusive Design Research Centre"],
+	"licenses": [{
+		"url": "https://creativecommons.org/licenses/by/4.0/",
+		"name": "Creative Commons Attribution 4.0 International Public License",
+		"version": "4.0"
+	}],
+	"sources": [{
+		"name": "Government of Ontario - Data Catalogue",
+		"web": "https://data.ontario.ca/dataset/covid-19-assessment-centre-locations"
+	}, {
+		"name": "WeCount Project - Inclusive Design Research Centre",
+		"web": "https://wecount.inclusivedesign.ca/"
```

Comment:
I think we shouldn't add as a source something which isn't directly or indirectly a location from which we could acquire the data. Probably there should be no sources listed here since this data is self-sourced.